x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -328,6 +334,16 @@ class PackageClientImpl implements PackageClient {
     return getPackageFieldsMetadata(params, options);
   }
 
+  public async getLatestPackageInfo(packageName: string, prerelease?: boolean) {
```

Comment:
it would be great to add `packageVersion` parameter for consistency

Reply:
I understand the desire for consistency, but our use-case explicitly was to get the info of the latest version without knowing what that is. Hence I included the `Latest` in the method name. What would your preferred way of handling that use-case be?

Reply:
You're right, `Latest` indicates that it gets the latest version, then it's good as is.